bluebottle/payments_flutterwave/adapters.py

```diff
@@ -18,15 +20,44 @@
 
 
 class FlutterwavePaymentAdapter(BasePaymentAdapter):
-    MODEL_CLASSES = [FlutterwavePayment]
+    MODEL_CLASSES = [FlutterwavePayment, FlutterwaveMpesaPayment]
```

Comment:
Wouldn't it be a little simpler if we had one model per adapter, and have both the MPesa en the Verve implementation of flutterwave extend from a common base adapter?